README.md

```diff
@@ -52,14 +52,37 @@ print("Number of entities found: ", len(entities))
 
 ## Installation
 
-Ner4Opt requires Python 3.X+ and can be installed from PyPI using `pip install ner4opt` or by building from source 
+Ner4Opt requires Python 3.9+ and can be installed from PyPI using `pip install ner4opt` or by building from source 
```

Comment:
learning question; why 3.9+ strictly? why not 8 or 7?

Reply:
Testing with 8 and 7 should work.

Reply:
Works with 8 and above. I will have to retrain sklearn models for 7 and below.